client/java/src/main/java/com/linecorp/centraldogma/client/Watcher.java

```diff
@@ -133,6 +133,13 @@ default Latest<T> awaitInitialValue(long timeout, TimeUnit unit) throws Interrup
         requireNonNull(unit, "unit");
         try {
             return initialValueFuture().get(timeout, unit);
+        } catch (TimeoutException e) {
+            final long initialTimeoutMillis = TimeUnit.MILLISECONDS.convert(timeout, unit);
+            throw new TimeoutException("Failed to get the initial value in " + initialTimeoutMillis + " ms. " +
+                                       "It's probably because the timeout value is too small or " +
+                                       "the target entry doesn't exist. Please consider using " +
+                                       "'errorOnEntryNotFound option' to get " +
+                                       EntryNotFoundException.class.getSimpleName() + " for that case.");
```

Comment:
Since `that case` is ambiguous

```suggestion
            throw new TimeoutException("Failed to get the initial value in " + initialTimeoutMillis + " ms. " +
                                       "It's probably because the timeout value is too small or " +
                                       "the target entry doesn't exist. Please consider using " +
                                       "the 'errorOnEntryNotFound' option to get a " +
                                       EntryNotFoundException.class.getSimpleName() + " for a missing entry.");
```